email-service/src/main/resources/application.conf

```diff
@@ -32,9 +32,9 @@ aws {
 hmda {
   lar {
     email {
-      subject = "You Filed your HMDA data!"
+      subject = "HMDA filing accepted!"
       subject = ${?EMAIL_SUBJECT}
-      content = "Your submission has been processed"
+      content = "<b>HMDA filing accepted!</b><br>Congratulations, you have successfully completed your HMDA filing for <period>!<br>Your data and signature were received and recorded on <datetime>. Your receipt number for this submission is <uniq_id> "
```

Comment:
Should we change this in the `values.yaml` file/configmap as well or do we just want to remove that piece and not override what's here?

Reply:
@kgudel addressed

Reply:
👍